clippy-book/book.toml

```diff
@@ -0,0 +1,7 @@
+[book]
+authors = ["Moaz bin Mohamed Mokhtar"]
```

Comment:
Please use the same author we use in all the other Cargo.toml files

Reply:
I'm confused because I found many Cargo.toml with different authors as below. What is the advise?
**Cargo.toml in main directory**
```toml
authors = [
	"Manish Goregaokar <manishsmail@gmail.com>",
	"Andre Bogus <bogusandre@gmail.com>",
	"Georg Brandl <georg@python.org>",
	"Martin Carton <cartonmartin@gmail.com>",
	"Oliver Schneider <clippy-iethah7aipeen8neex1a@oli-obk.de>"
]
```
**Cargo.toml in clippy_dev**
```toml
authors = ["Philipp Hansch <dev@phansch.net>"]

```
**Cargo.toml in clippy_dummy**
```toml
authors = ["Manish Goregaokar <manishsmail@gmail.com>"]

```
**Cargo.toml in clippy_lints**
```toml
authors = [
	"Manish Goregaokar <manishsmail@gmail.com>",
	"Andre Bogus <bogusandre@gmail.com>",
	"Georg Brandl <georg@python.org>",
	"Martin Carton <cartonmartin@gmail.com>"
]
```